### ISSUE TITLE
No reply button on pages feed to reply to a pages forum thread post

### ISSUE THREAD

**Issue by prepperz** [PHPfox-Official/phpfox-v4-issues#2536]
## What happened?
1. No reply button on pages feed to reply to a pages forum thread post
2. Multi line thread post show as single in feed.
3. Not intuitive or user friendly at all. 

## Steps to reproduce:
1. post a multi-line reply in a pages forum thread 

## What is expected?
Pages feed and thread forum feels disconnected and disorienting to each other. The transition is not smooth.

## Browsers and Devices tested
Chrome on Windown 10

## Server information
24 Cores, CentOS 7.6.1810, php7.0.27, apache 2.4.6, MySQL: 5.5.60-MariaDB

...

## phpFox version
phpFox 4.7.2 (Upgraded), Material theme

...

**Comment by harrison05** [comment]
1. Currently, we don't support reply thread on a feed of new thread.
2. Let we check it, post's content was strip all tags on feed

Regards.

**Comment by prepperz** [comment]
ya. It just feels disconnected and looks terrible.

**Comment by harrison05** [comment]
Yes, we're working on Forums to improve its feed and allow a user can reply thread on feed. Please wait for a new version.

Regards.

**Comment by PhpFoxJohnJr** [comment]
Was this fixed?
